### PR TITLE
wireless devices autoconnection

### PR DESCRIPTION
**Pull request by red-avtovo**
added daemon for wireless device autoconnection

### CODE REVIEW

File: Appium/wireless_connect.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if [ ! -z "$ANDROID_DEVICES" ]; then
+    connected_devices=$(adb devices)
+    IFS=',' read -r -a array <<< "$ANDROID_DEVICES"
+    for i in "${!array[@]}"
+    do
+        array_device=$(echo ${array[$i]} | tr -d " ")
+        #string contains check
+        if [[ ${connected_devices} != *${array_device}* ]]; then
```
Comment (mykola-mokhnach) [inline Appium/wireless_connect.sh]: wouldn't it be more appropriate to user grep for this purpose?

File: Appium/wireless_connect.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+            echo "Connecting to: ${array_device}"
+            adb connect ${array_device}
```
Comment (mykola-mokhnach) [inline Appium/wireless_connect.sh]: should we check the exit code here to make sure the connection was really successful?

Comment (red-avtovo) [reply Appium/wireless_connect.sh]: No, we don't care. Anyway script will be run again after specified delay.
It is the main idea of this PR. Script will try to reconnect to remote device in any case

**Comment by red-avtovo** [comment]
Unfortunately this PR is not finished

**Comment by SrinivasanTarget** [comment]
Awesome @red-avtovo 👍

**Comment by red-avtovo** [comment]
Now it's done!

**Pull request merged**